vimeo/VimeoUpload
# VIM-7293: Fixes use of Vimeo response serializer based VN refactor

#### Ticket

[VIM-7293](https://vimean.atlassian.net/browse/VIM-7293)

#### Ticket Summary

See [this PR](https://github.com/vimeo/VimeoNetworking/pull/359) for details

#### Implementation Summary

* Refactor use of VimeoResponseSerializer to match new API on VimeoNetworking
* Removes force unwrapping in favor of throwing an error

#### How to Test

- Build and run, ensure everything looks and works as expected
- Make sure you are able to download videos for offline viewing, and subsequently play them.
- Make sure you are able to upload videos, and subsequently play them.
